editor/editor_inspector.cpp

```diff
@@ -4742,6 +4743,15 @@ void EditorInspector::_notification(int p_what) {
 				update_tree();
 			}
 		} break;
+
+		case NOTIFICATION_FOCUS_ENTER: {
+			set_deferred("follow_focus", true);
```

Comment:
Is `deferring` needed here?

Reply:
I had problems without deferring, but it might've been caused by another thing I fixed. I'll check again to make sure.